memcached-mixin/alerts.libsonnet

```diff
@@ -17,6 +17,8 @@
               message: |||
                 Memcached Instance {{ $labels.job }} / {{ $labels.instance }} is down for more than 15mins.
               |||,
+              description: 'Memcached Instance {{ $labels.job }} / {{ $labels.instance }} is down for more than 15mins.',
```

Comment:
I think that message should be removed if we have description and summary instead.

Reply:
+1, especially as the linter ignores 'message', so the format could end up wonky

Reply:
Thanks! Removed messages